src/libxtp/threecenter_gwbse.cc

```diff
@@ -149,7 +187,25 @@ void TCMatrix_gwbse::FillBlock(std::vector<Eigen::MatrixXd>& block,
       }
     }  // gamma-loop
   }    // alpha-loop
-  for (int k = 0; k < auxshell.getNumFunc(); ++k) {
+
+  return symmstorage;
+}
+
+/*
+ *  Convolution of the GW shell with ALL functions with the DFT orbital
+ * coefficients
+ */
+void TCMatrix_gwbse::FillBlock(std::vector<Eigen::MatrixXd>& block,
```

Comment:
This is not a requirement, but the `std::vector<Eigen::MatrixXd>block` should be the return type instead of `void`. But that is a general thing.

Reply:
done